beastify/popup/choose_beast.css

```diff
@@ -2,15 +2,31 @@ html, body {
   width: 100px;
 }
 
-.beast {
+.button {
   margin: 3% auto;
   padding: 4px;
   text-align: center;
   font-size: 1.5em;
-  background-color: #E5F2F2;
   cursor: pointer;
 }
 
-.beast:hover {
+.button:hover {
```

Comment:
I think this should be "beast"? "button" works, because it gets overridden by "clear", but it seems like we really only want to apply this to beasts, not to all buttons.